stencil-library/src/components/payment-details/payment-details.tsx

```diff
@@ -107,6 +102,9 @@ export class PaymentDetails {
                 <DetailItem title="Brand" value={this.payment.payment_method.card.brand} />
                 <DetailItem title="Cardholder" value={this.payment.payment_method.card.name} />
               </DetailSection>
+              <DetailSection sectionTitle='Metadata'>
```

Comment:
@colganmeanor I think in this component we should add the `DetailsSection` parts: 
`detail-section` and `details-section-title`